app/message-attachments/client/messageAttachment.html

```diff
@@ -151,6 +151,23 @@
 				{{/unless}}
 			{{/if}}
 
+			{{#with uploading}}
+				<div class="attachment-upload-progress color-primary-action-color background-component-color {{#if error}}error-background error-border{{/if}}">
```

Comment:
Please provide a screenshot of this UI change.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/17441#issuecomment-671018953
The Progress bar below the attachment, within message.